presentation/src/main/java/com/mashup/gabbangzip/sharedalbum/presentation/ui/invitation/InvitationCodeViewModel.kt

```diff
@@ -39,9 +41,24 @@ class InvitationCodeViewModel @Inject constructor(
                         state.copy(
                             isLoading = false,
                             isInvitationSuccessful = false,
+                            errorMessageRes = mapErrorMessageRes(e),
                         )
                     }
                 }
         }
     }
+
+    private fun mapErrorMessageRes(e: Throwable): Int {
+        return when (e) {
+            is PicException.InvalidGroupCodeException -> {
+                R.string.enter_group_by_code_failure_not_found
+            }
+
+            is PicException.GroupOverflowException -> {
+                R.string.enter_group_by_code_failure_overflow
+            }
+
+            else -> R.string.error_network
+        }
```

Comment:
```suggestion
        return when (e) {
            is PicException.InvalidGroupCodeException -> R.string.enter_group_by_code_failure_not_found
            is PicException.GroupOverflowException -> R.string.enter_group_by_code_failure_overflow
            else -> R.string.error_network
        }
```
이건 진짜진짜진짜 사소하긴 한데...
다 한줄에 표현 가능해보여서 괄호 없이 붙여줘도 괜찮을지도..?

Reply:
[2b47e90](https://github.com/mash-up-kr/gabbangzip-Android/pull/324/commits/2b47e901814299e27318a191a9a11f541823cd90) 반영완!